### PR TITLE
fix(license) : Refactoring code due to license issue

### PR DESCRIPTION
**Pull request by mynktl**
Changes: 
- Moving test_uzfs.sh to openebs/libcstor
- Removing zpios related code which includes `cmd/zpios` and `module/zpios`
- Moving `config/ax_code_coverage.m4` to `tests/cbtest/ax_code_coverage.m4`

Signed-off-by: mayank <mayank.patel@mayadata.io>

### CODE REVIEW

**Review by vishnuitta — APPROVED** [review]
changes are good

**Comment by mynktl** [comment]
This PR has dependency on https://github.com/openebs/libcstor/pull/18.
Once https://github.com/openebs/libcstor/pull/18 is merged, i will restart travis build.

File: autogen.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-autoreconf -fiv
+autoreconf -fiv --include tests/cbtest
```
Comment (vishnuitta) [inline autogen.sh]: @mynktl we need to change cbtest to cstor here as well

**Review by vishnuitta — CHANGES_REQUESTED** [review]
change 'cbtest' to 'cstor'

File: autogen.sh (hunk 1 above)
Comment (mynktl) [reply autogen.sh]: ok

**Comment by mynktl** [comment]
This PR has a dependency on openebs/libcstor#19.
Once openebs/libcstor#19 is merged, I will restart Travis build.

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**